BizHawk.Emulation.Cores/Consoles/Nintendo/Gameboy/Gambatte.cs

```diff
@@ -344,14 +344,16 @@ internal void FrameAdvancePrep(IController controller)
 			LibGambatte.gambatte_setlayers(GambatteState, (_settings.DisplayBG ? 1 : 0) | (_settings.DisplayOBJ ? 2 : 0) | (_settings.DisplayWindow ? 4 : 0));
 		}
 
-		internal void FrameAdvancePost()
+		internal bool FrameAdvancePost()

```

Comment:
Since the gambatte internal code doesn't actually do any of this stuff (although it could easily be made to), would it be more appropriate to leave this method alone and only make the `return true;` on the `FrameAdvance` method?

Reply:
Oops, that's my bad. Yeah it should just be void.